setup.py

```diff
@@ -0,0 +1,24 @@
+from setuptools import setup, find_packages
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setup(
+    name='aliyunstoreplugin',
+    version='1.0.0',
+    description='Plugin that provides Aliyun oss Artifact Store functionality for MLflow',
+    long_description=long_description,
+    long_description_content_type="text/markdown",
+    author='Xianyan Jia',
+    author_email='jiaxianyan@gmail.com',
+    url="https://github.com/SeaOfOcean/mlflow-aliyunstore",
+    packages=find_packages(),
+    install_requires=[
+        'mlflow',
```

Comment:
@SeaOfOcean Can we add `oss2` as a required installation dependency here?

Reply:
thanks, added `oss2` as a dependency